nearcore/src/metrics.rs

```diff
@@ -0,0 +1,22 @@
+use near_metrics::{try_create_histogram_vec, HistogramVec};
+use once_cell::sync::Lazy;
+
+pub static APPLY_CHUNK_DELAY: Lazy<HistogramVec> = Lazy::new(|| {
+    try_create_histogram_vec(
+        "near_apply_chunk_delay",
+        "Time process a chunk. Tgas used by the chunk is a metric label.",
+        &["tgas_ceiling"],
+        Some(prometheus::linear_buckets(0.0, 0.05, 50).unwrap()),
+    )
+    .unwrap()
+});
+
+pub static SECS_PER_THOUSAND_TGAS: Lazy<HistogramVec> = Lazy::new(|| {
+    try_create_histogram_vec(
+        "near_execution_secs_per_thousand_tgas",
+        "Execution time per teragas. Ignore label 'label'.",
+        &["label"],
+        Some(prometheus::linear_buckets(0.0, 0.05, 50).unwrap()),
```

Comment:
Is linear the best bucket distribution to use here? I suspect more bucket density around 1 second would help investigating problems around situations where the relationship doesn't hold just barely, and once we're looking at numbers like 2 seconds or more per Pgas, it is going to be pretty obvious things are bad no matter what, right?

Reply:
Done